src/app/static/src/app/store/adr/actions.ts

```diff
@@ -143,20 +144,28 @@ export const actions: ActionTree<ADRState, RootState> & ADRActions = {
         }
     },
 
-    async uploadFilestoADR(context, uploadFilesPayload) {
+    async uploadFilesToADR(context, uploadFilesPayload) {
         const {state, rootState, commit} = context;
+        const uploadMetadata = rootState.modelCalibrate.uploadMetadata
         const selectedDatasetId = rootState.baseline.selectedDataset?.id;
         const modelCalibrateId = rootState.modelCalibrate.calibrateId;
         commit({type: ADRMutation.ADRUploadStarted});
 
         for (let i = 0; i < uploadFilesPayload.length; i++) {
-            const { resourceType, resourceFilename, resourceId } = uploadFilesPayload[i]
+            const { resourceType, resourceFilename, resourceId, displayName } = uploadFilesPayload[i]
 
             const requestParams: Dict<string> = {resourceFileName: resourceFilename}
             if (resourceId){
                 requestParams["resourceId"] = resourceId
             }
 
+            if (displayName === "outputSummary") {
+                requestParams["description"] = uploadMetadata.outputSummary.description
```

Comment:
One issue with this approach that occurs to me is that it's not backward compatible - this will fail for any saved projects whose calibrate results were generated before this feature went in. I can imagine users wanting to upload files to ADR from existing projects. I guess we can tell them that they'll need to recalibrate in order to that? But in that case we should at least fail nicely / show them a meaningful error message which explains the problem. Any thoughts on that @r-ash ?

Reply:
I missed this, and my hunch is that we should prompt users to recalibrate rather than unilaterally upload some placeholder text that is ultimately inconsistent with the format/content of whatever we get from Naomi (which I know is still TBC). But agree that @r-ash should pass judgement here!

Reply:
Agreed, should definitely not be uploading a placeholder, just need to check that prompting for recalibrate is acceptable - I think it should be as calibration is relatively quick, so that probably makes more sense than putting in some special workaround like extra endpoint just to fetch the description.